src/scenes/home/informationForm/informationForm.js

```diff
@@ -68,9 +81,24 @@ class SignupInformation extends Component {
     this.setState({ step: this.state.step -= 1 });
   }
 
+  isFormValid = () => {
+    let valid = true;
+    if (!this.state.isDescriptionValid) {
+      valid = false;
+    }
+
+    this.setState({ error: !valid });
+    return valid;
```

Comment:
Would we be able to remove the `valid` variable in favor of `this.state.isDescriptionValid`? For example:
```javascript
isFormValid()  {
  this.setState({ error: this.state.isDescriptionValid });
  return this.state.isDescriptionValid;
}

Reply:
Certainly we would.
When I wrote this function I was thinking about how we might have to use it in the future for validating other components. In that case I think it might be easier to have a variable which will be later assigned to the `error` key of the state.